cmd/deploy/remote.go

```diff
@@ -223,22 +223,23 @@ func (rd *remoteDeployCommand) createDockerfile(tmpDir string, opts *Options) (s
 }
 
 func (rd *remoteDeployCommand) createDockerignoreIfNeeded(cwd, tmpDir string) error {
```

Comment:
Can we add a test for this function? We should check that we create the right file with the content without leaking the content from the `.dockerignore`

Reply:
done b91e64af29e071cbc55246556b66f024435f2ea3